PURL-TYPES.rst

```diff
@@ -397,6 +397,59 @@ nuget
 
       pkg:nuget/EnterpriseLibrary.Common@6.0.1304
 
+p2
+----
+``p2`` for Eclipse p2 units:
+
+- A p2 unit represents a unit of software that can be consumed using the Eclipse
+  Provisioning Platform. A unit can represent various types of software 
+  artifacts, such as Eclipse features, plug-ins or other bundles.
+
+- A p2 unit is uniquely identified by its id and version. A namespace is
+  therefore not necessary.
+
+- The name corresponds to the unique identifier of a given p2 unit. This id
+  uniquely identifies a unit within the Eclipse Provisioning Platform. 
+  Examples::
+
+      org.eclipse.core.resources.win32.x86_64
+      org.eclipse.rcp.sdk.id.executable.win32.win32.x86_64
+      org.eclipse.sdk.feature.group
+
+- The version corresponds to the version of the p2 unit. A p2 repository may
+  host multiple versions of the same unit. This information is therefore
+  essential when resolving the relationship between different artifacts.
+  Examples::
+
+      3.5.500.v20220812-1420
+      2.0.0.202304281106
+
+- The software artifact are accessed from a p2 repository. Given that each
```

Comment:
```suggestion
- The software artifact are accessed from a p2 artifact repository. Given that each
```

While it's common to have both artifacts and metadata repository at the same location, one could have both split, and metadata repo is not relevant here.